backend/database/ontology/SDG_Ontology.owl

```diff
@@ -111,8 +119,8 @@
         <rdfs:domain rdf:resource="http://www.semanticweb.org/aga/ontologies/2017/9/SDG#Approaches"/>
         <rdfs:domain rdf:resource="http://www.semanticweb.org/aga/ontologies/2017/9/SDG#Metoder"/>
         <rdfs:range rdf:resource="http://www.semanticweb.org/aga/ontologies/2017/9/SDG#Kommuner"/>
-        <SDG:description rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Binære relasjon som beskriver at en klasse/instanse (av byer/tettsted) som har en klasse/instanse (av metoder) som er gjennomført eller gjennomførers denne klassen/instansen</SDG:description>
```

Comment:
I'd prefer that we don't merge these changes (in this file), as they strip type information from fields.